rest-server/src/main/java/io/github/nubesgen/configuration/NubesgenConfiguration.java

```diff
@@ -228,13 +242,20 @@ public boolean isRuntimeMaven() {
             RuntimeType.SPRING.equals(this.getRuntimeType()) ||
             RuntimeType.QUARKUS.equals(this.getRuntimeType()) ||
             RuntimeType.QUARKUS_NATIVE.equals(this.getRuntimeType()) ||
+            RuntimeType.MICRONAUT.equals(this.getRuntimeType()) ||
```

Comment:
should not this be better as a method of `RuntimeType`. e.g. `RuntimeType::Maven?`